webapp/channels/src/sass/layout/_sidebar-left.scss

```diff
@@ -575,6 +575,19 @@ $sidebarOpacityAnimationDuration: 0.15s;
         }
     }
 
+    @media screen and (min-width: 768px) {
+        .SidebarNavContainer {
```

Comment:
With this change the menu can overlap the lhs scrollbar and doesn't need the property `position: fixed`, and these styles only apply when the lhs menu is shown sticky to the left (not by clicking the hamburguer menu)